src/pages/basic_data/app_admin/central_import/CentralImportRegistration.tsx

```diff
@@ -0,0 +1,35 @@
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@tanstack/react-query';
+import { createRegistrationFromImportCandidate, fetchImportCandidate } from '../../../../api/registrationApi';
+import { useTranslation } from 'react-i18next';
+import { Button } from '@mui/material';
```

Comment:
Vi bruker å ha importering av eksterne pakker øverst, og våre ting (alle imports som starter med `../`) etterpå.
Småpirk som ikke har noen betydning, men som jeg syns gir litt bedre oversikt
```suggestion
import { useTranslation } from 'react-i18next';
import { Button } from '@mui/material';
import { createRegistrationFromImportCandidate, fetchImportCandidate } from '../../../../api/registrationApi';
```